models/incident.js

```diff
@@ -48,6 +48,16 @@ schema.post('save', function() {
   schema.emit('incident:save', {_id: this._id.toString()});
 });
 
+schema.post('remove', function() {
+  Report.find({_incident:this._id.toString()}, function(err,reports){
```

Comment:
Makes sense, also checking this for more detail https://github.com/airbnb/javascript